test/hotspot/gtest/utilities/test_unsigned5.cpp

```diff
@@ -22,6 +22,7 @@
  */
 
 #include "precompiled.hpp"
+#include "runtime/os.hpp"
```

Comment:
Nit: please sort includes alphabetically

Reply:
Updated.  Thanks for the suggestion.